nodemon.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env":{
+        "JWT_KEY":"secret"
```

Comment:
Do not push your environment variables to github, use `.env` library to manage them.